discovery/eureka/metadata_marshaller.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2016 The Prometheus Authors
```

Comment:
nit, the copyright year for files you're adding should be 2020

Reply:
I will change it.